apps/backend/src/adoption/adoption.controller.ts

```diff
@@ -36,7 +39,9 @@ export class AdoptionsController {
   }
 
   @Patch()
+  @UseGuards(AccessTokenGuard, AccessGuard)
   @UsePipes(new JoiValidationPipe(adoptionValidationSchema))
+  @UseAbility(Actions.update, 'Adoption' as any, AdoptionHook)
```

Comment:
Had to cast this in purpose probably due to this issue https://github.com/getjerry/nest-casl/issues/349

Reply:
I see we use 'Adoption' in another place of the code, what do you think about having that separately on an enum or something?

Reply:
Yep, nice catch! I'll centralize and reuse it. Thanks